### ISSUE TITLE
cloudwand doesn't work when standing on ladder

### ISSUE THREAD

**Issue by VorTechnix** [sbrl/Minetest-WorldEditAdditions#66]
It selects the ladder segment by the players head (if there is one there). This is might be a ray cast issue.

**Comment by VorTechnix** [comment]
This is definitely a raycast issue. `//scloud` works fine under same conditions.

**Comment by sbrl** [comment]
Yeah, this is known bug in our raycaster. It's because `worldeditadditions.raycast` considers all nodes as full blocks. Fixing this would definitely be helpful, but it's also very complicated. To start on this, we need a way to:

1. Determine if a given node id has a custom hitbox that needs considering
2. If so, then retrieve said hitbox in world co-ordinates.
